src/components/layout/DefaultLayout.vue

```diff
@@ -58,6 +71,15 @@ export default defineComponent({
     }
   },
 
+  setup: () => {
+    const {signIn, userIsAuthenticated} = useAuth()
+
```

Comment:
We could actually add something like the following here:
```typescript
    const signInWithRedirect = () => {
      const currentFullPath = window.location.pathname + window.location.search + window.location.hash
      localStorage.setItem('redirectAfterLogin', currentFullPath)
      signIn()
    }
```

Then if we use `signInWithRedirect` above in the login hint, we will redirect users back to the current page.